packages/material-ui/src/useTouchRipple/useTouchRipple.ts

```diff
@@ -108,49 +113,54 @@ const useTouchRipple = (props: UseTouchRippleProps) => {
 
   const enableTouchRipple = mountedState && !disableRipple && !disabled;
 
-  const getRippleHandlers = React.useMemo(() => {
-    const rippleHandlers = {
-      onBlur: handleBlur,
-      onKeyDown: handleKeyDown,
-      onKeyUp: handleKeyUp,
-      onMouseDown: handleMouseDown,
-      onMouseUp: handleMouseUp,
-      onMouseLeave: handleMouseLeave,
-      onContextMenu: handleContextMenu,
-      onDragLeave: handleDragLeave,
-      onTouchStart: handleTouchStart,
-      onTouchEnd: handleTouchEnd,
-      onTouchMove: handleTouchMove,
-    } as RippleEventHandlers;
```

Comment:
cast was not needed. `rippleHandlers: RippleEventHandlers` would've been sufficient.